CodeCoverageQualityGate/BuildApiClient.ts

```diff
@@ -0,0 +1,188 @@
+/*
```

Comment:
I see this is almost the same as BuildApiClient.ts in the other task. Did we research how to share common code across different tasks in ADO?